src/components/workspace/side-content/index.tsx

```diff
@@ -18,31 +18,28 @@ class SideContent extends React.PureComponent<SideContentProps, {}> {
     return (
       <div className="side-content">
         <Card>
-          {this.renderHeader()}
-          <div className="side-content-text">{this.props.tabs[this.props.activeTab].body}</div>
+          <Tabs
+            id="side-content-tabs-container"
+            defaultSelectedTabId={0}
+            selectedTabId={this.props.activeTab}
+            onChange={this.onChangeTab}
+          >
+            {this.props.tabs.map((tab, key) => (
+              <Tab
+                id={key}
+                key={key}
+                title={tab.label}
+                panel={<div className="side-content-text">{tab.body}</div>}
+              />
+            ))}
+          </Tabs>
```

Comment:
Can you revert the commits making this a blueprintjs tab? I meant only to change the implementation, without changing the look, i.e.

- Different tabs are still indicated by icons
- The icons are still centered
- There is still a visual divide between icons and side content content. It's just an awkward empty space right now.